### ISSUE TITLE
Helm get symbols timeout to short

### ISSUE THREAD

**Issue by ihdavids** [OmniSharp/omnisharp-emacs#355]
Any chance we can make a global default for the timeout-seconds in here in omnisharp-utils? 
Or provide a means of piping a timeout value through in the symbols-candidates method?

In any project of reasonable size I find I am bumping into the timeout here when using 
omnisharp-helm-find-symbols:

omnisharp-helm-find-symbols
    omnisharp--helm-find-symbols-candidates
         omnisharp--send-command-to-server-sync
              >> omnisharp--wait-until-request-completed (timeout happens in here)

Here is the actual timeout:

(defun omnisharp--wait-until-request-completed (request-id
                                                &optional timeout-seconds)
  (setq timeout-seconds (or timeout-seconds 2))

**Comment by razzmatazz** [comment]
Yes we can fix this fairly easily I think. The underlying issue is in
omnisharp Roslyn server where the implementation is a little bit naive.
Increasing the timeout should be a quick fix for now..

ihdavids <notifications@github.com> schrieb am Di., 15. Aug. 2017, 17:43:

**Comment by razzmatazz** [comment]
@ihdavids btw, what kind of timeout would you seem reasonable? have experimented to check which timeout value works for you best?
